compiler/src/dotty/tools/backend/jvm/DottyBackendInterface.scala

```diff
@@ -24,7 +24,7 @@ import StdNames.nme
 import NameKinds.LazyBitMapName
 import Names.Name
 
-class DottyBackendInterface(val outputDirectory: AbstractFile, val superCallsMap: ReadOnlyMap[Symbol, Set[ClassSymbol]])(using val ctx: Context) {
+class DottyBackendInterface(val superCallsMap: ReadOnlyMap[Symbol, Set[ClassSymbol]])(using var ctx: Context) {
```

Comment:
I've changed it to var because now we use the instance of CodeGen between multiple runs we need to replace the current compilation unit within the context. It's required e.g. to calculate correct source positions. I've reverted it to use immutable val while modifying it using underlying `FreshContext.setCompilationUnit`